css/style.css

```diff
@@ -0,0 +1,44 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+#main{
+    background-color: pink;
+    width: 80%;
+    margin: auto;
+    padding: 1rem;
+}
+#title,p{
+    text-align: center;
+    margin: 0.5rem 0;
+}
+
+#image{
+    max-width: 100%;
+    height: auto;
+    display: block;
+    border-radius: 5px;
+    margin: 1rem 0;
+}
+#tribute-link{
+    display: block;
+    width: 40%;
+    margin: 1rem auto;
+    text-decoration: none;
+    color: rgb(41, 202, 27);
+    padding: 1rem;
+    background-color: aliceblue;
+    border-radius: 3px;
+}
+/*
+.timeline{
+   
+}*/
```

Comment:
Please do not leave unused code in your codebase.  
You can erase from line 35 till line 38.

Reply:
Nice job being polite and giving reasons why you requested changes in your previous comments😉. Just as you have done previously, kindly state the reason why you are requesting a change😉. This makes your comments clearer to the recipient👍.
<img width="581" alt="image" src="https://github.com/IvonneBenitesRodriguez/reviewer_project/assets/64643995/05f143ca-aa7f-4c45-af78-12abbef30662">

You can check this[ lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for a refresher😉.

Reply:
- [ ] As highlighted by the previous reviewer, kindly give reasons for requesting a change to the code base. Clear and specific reasons help the author of the code to understand the intent behind the suggested changes, fostering better communication and reducing ambiguity. This allows for a more constructive dialogue between team members. 👍 

#### Hint: Edit your comment above and give reasons why there is a need to remove unused code in the codebase. 😎